lib/config/file.go

```diff
@@ -57,13 +57,13 @@ func NewNamespacesFromDir(nsdir string) (*NamespaceDir, error) {
 			return nil, err
 		}
 		c.cfgs[cfg.Namespace] = &cfg
-		c.nspath[cfg.Namespace] = yamlFile
+		c.nspath[cfg.Namespace] = nfile
 	}
 
 	return c, nil
 }
 
-func listAllYamlFiles(dir string) ([]string, error) {
+func listAllFiles(dir, ext string) ([]string, error) {
```

Comment:
Isn't it duplicate with
https://github.com/pingcap/TiProxy/blob/dc6452506016dbcdba179d7253555b14ae5e117d/lib/cli/namespace.go#L36

Reply:
Correct, I am removing `/lib/config/file`.